openIMIS/openIMIS/settings.py

```diff
@@ -200,22 +202,32 @@ def SITE_ROOT():
         'extra_params': "Persist Security Info=False;server=%s" % os.environ.get('DB_HOST'),
         'unicode_results': True
     }
-else:
+elif not os.environ.get('SIMPLE_DATABASE', False):
     DATABASE_OPTIONS = {
         'driver': 'ODBC Driver 17 for SQL Server',
         'unicode_results': True
     }
 
-DATABASES = {
-    'default': {
-        'ENGINE': os.environ.get('DB_ENGINE', 'sql_server.pyodbc'),
-        'NAME': os.environ.get('DB_NAME'),
-        'USER': os.environ.get('DB_USER'),
-        'PASSWORD': os.environ.get('DB_PASSWORD'),
-        'HOST': os.environ.get('DB_HOST'),
-        'PORT': os.environ.get('DB_PORT'),
-        'OPTIONS': DATABASE_OPTIONS}
-}
+
+if os.environ.get('SIMPLE_DATABASE', 'False') == 'True':
+    # used by AI instance that does not require running database
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        }
+    }
+else:
+    DATABASES = {
```

Comment:
Use NO_DATABASE_ENGINE env var to disable DB.

Reply:
NO_DATABASE_ENGINE used